### DISCUSSION TITLE
[Feature Request] Add sample for integrating monaco editor into browser extension's content script

### DISCUSSION THREAD

**Discussion by yuumi-the-developer** [microsoft/monaco-editor#3881] [Q&A]
### Context

- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*
- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*

### Proposal

I propose adding a new sample in the `samples` directory of the Monaco Editor repository that demonstrates how to integrate the Monaco Editor into browser extension's content script. As it's quite big, I think we have to inject it with executeScript. 

### Benefits

Having a sample of that would:

- Provide a clear starting point for developers looking to use the Monaco Editor in their browser extension projects.
- Increase the visibility and potential use cases for the Monaco Editor.

### The main reason

I don't know how to do it myself.

**Comment by hediet** [comment]
We are open to link to an external demo, but I don't think such a demo should be included in this repository, as we would need to maintain it.

**Reply by yuumi-the-developer** [reply]
I did it, I did it!
https://github.com/yuumi-the-developer/content-script-monaco

**Reply by yuumi-the-developer** [reply]
I don't know if I did it right tho 😔

**Reply by yuumi-the-developer** [reply]
But it's working 😃
